packages/dx-react-grid/README.md

```diff
@@ -12,21 +12,27 @@ The DevExtreme React Grid component allows you to display table data with a set
 npm i @devexpress/dx-react-grid --save
 ```
 
-This package does not contain any visual components, so all examples provided below use the DevExtreme React Grid Bootstrap3 package to use Bootstrap rendering for the Grid visual components.
+This package does not contain any visual components; all examples below use the DevExtreme React Grid Bootstrap3 package to use Bootstrap rendering for the Grid visual components.
```

Comment:
"all examples below _use_ the DevExtreme React Grid Bootstrap3 package to _use_ Bootstrap rendering for the Grid visual components" is a confusing sentence. Consider rewriting it.